### ISSUE TITLE
Apache plugin show notification when parsing configuration file with multiple VirtualHosts

### ISSUE THREAD

**Issue by joohoi** [certbot/certbot#2856]
Apache plugin should notify the user when parsing a file with multiple VirtualHosts, currently we aren't telling the user what we do expect of the configuration schema. The current functionality - to just leave user with a message "No virtualhost found" is not sufficient to identify the reason why this happened.

This issue was originally reported in #2776

**Comment by geerlingguy** [comment]
I just ran into this issue when attempting to acquire a few certs for a server where all the vhosts are defined in one .conf file.

I ran:

```
./certbot-auto --apache certonly -d www.example.com
```

Then I chose my account, and then got the following message:

```
We were unable to find a vhost with a ServerName or Address of
www.example.com.
Which virtual host would you like to choose?
(note: conf files with multiple vhosts are not yet supported)
```

The files listed didn't include the vhosts.conf file that had all the vhosts (but it did include other vhost conf files that defined only one virtualhost).

I selected 'Cancel' on that screen... but then certbot spent a little time doing something, and spat out:

```
IMPORTANT NOTES:
 - Congratulations! Your certificate and chain have been saved at
   /etc/letsencrypt/live/www.example.com/fullchain.pem. Your cert
   will expire on 2016-10-09. To obtain a new or tweaked version of
   this certificate in the future, simply run certbot-auto again. To
   non-interactively renew *all* of your certificates, run
   "certbot-auto renew"
 - If you like Certbot, please consider supporting our work by:

   Donating to ISRG / Let's Encrypt:   https://letsencrypt.org/donate
   Donating to EFF:                    https://eff.org/donate-le
```

Testing out the cert (I manage my config manually), it seems to work fine. So it seems only the Apache vhost configuration part of Certbot's functionality doesn't work.

I passed in the `certonly` option—is there a way we could remove those scary warnings about the multiple vhost config problem if that option is present, since it is definitely working? (Site in question: https://www.ansiblefordevops.com/ — cert seems to be [green in my testing](https://www.ssllabs.com/ssltest/analyze.html?d=www.ansiblefordevops.com)).

**Comment by pde** [comment]
Closing this since we're about to land a fix for #1042 instead...
